Library/Homebrew/utils/bottles.rb

```diff
@@ -25,20 +25,20 @@ def file_outdated?(f, file)
       end
 
       def native_regex
-        /(\.#{Regexp.escape(tag.to_s)}\.bottle\.(\d+\.)?tar\.gz)$/o
+        /(\.#{Regexp.escape(tag.to_s)}\.bottle\.(\d+\.)?tar\.(?:gz|xz))$/o
       end
 
       def receipt_path(bottle_file)
-        path = Utils.popen_read("tar", "-tzf", bottle_file).lines.map(&:chomp).find do |line|
-          line =~ %r{.+/.+/INSTALL_RECEIPT.json}
-        end
+        command = tar_command "tf", bottle_file, "*/*/INSTALL_RECEIPT.json"
```

Comment:
`*/*/INSTALL_RECEIPT.json` requires the option `--wildcards` with GNU tar on Linux. I'd prefer to keep the previous code for compatibility with both BSD tar and GNU tar.
```
tar: Pattern matching characters used in file names
tar: Use --wildcards to enable pattern matching, or --no-wildcards to suppress this warning
tar: */*/INSTALL_RECEIPT.json: Not found in archive
```

Reply:
Which version of tar and on which host? My Fedora 27 has 1.29 and returns as expected.

Reply:
GNU tar 1.30 on Linuxbrew. For example…
```sh
❯❯❯ brew install gnu-tar
❯❯❯ brew fetch hello
❯❯❯ gtar -tf ~/Library/Caches/Homebrew/hello-2.10.high_sierra.bottle.tar.gz '*/*/INSTALL_RECEIPT.json'
gtar: Pattern matching characters used in file names
gtar: Use --wildcards to enable pattern matching, or --no-wildcards to suppress this warning
gtar: */*/INSTALL_RECEIPT.json: Not found in archive
gtar: Exiting with failure status due to previous errors
```